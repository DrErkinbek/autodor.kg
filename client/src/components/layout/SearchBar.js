import React from 'react';

const SearchBar = () => {
    return(
        <div className="row">
            <div className="col s12 m12 l12">
                <div className="nav-wrapper">
                    <form>
                        <div className="input-field">
                            <input id="search" type="search"
                            placeholder="Ищите по размеру" />
                                <label className="label-icon" htmlFor="search">
                                    <i className="material-icons">search</i>
                                </label>
                            <i className="material-icons">close</i>
                        </div>
                    </form>
                </div>
            </div>  
        </div>
    )
}

export default SearchBar;