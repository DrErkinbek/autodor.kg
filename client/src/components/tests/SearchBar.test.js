import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import SearchBar from '../layout/SearchBar';

it('SearchBar Component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SearchBar />, div);
});

it('SearchBar Component renders correctly and equal to snapshot', () => {
    const tree = renderer.create(<SearchBar />).toJSON();
  
    expect(tree).toMatchSnapshot();
});