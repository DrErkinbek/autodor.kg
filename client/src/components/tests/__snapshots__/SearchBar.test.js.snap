// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Component renders correctly and equal to snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col s12 m12 l12"
  >
    <div
      className="nav-wrapper"
    >
      <form>
        <div
          className="input-field"
        >
          <input
            id="search"
            placeholder="Ищите по размеру"
            type="search"
          />
          <label
            className="label-icon"
            htmlFor="search"
          >
            <i
              className="material-icons"
            >
              search
            </i>
          </label>
          <i
            className="material-icons"
          >
            close
          </i>
        </div>
      </form>
    </div>
  </div>
</div>
`;

exports[`Spinner Component renders correctly and equal to snapshot 1`] = `
<div
  className="container center"
>
  <div
    className="preloader-wrapper big active"
  >
    <div
      className="spinner-layer spinner-blue-only"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>
</div>
`;
